examples/bitcoinjs-lib/test/pretest.ts

```diff
@@ -8,6 +8,8 @@ async function pretest() {
     execSync(`dfx deploy`, {
         stdio: 'inherit'
     });
+
+    execSync(`cd ../bitcore-lib && npm install`, { stdio: 'inherit' });
```

Comment:
I think this deserves a comment explaining its existence

Reply:
I'm going to make an issue for this, please address it soon: https://github.com/demergent-labs/azle/issues/1745